.github/workflows/push-master.yaml

```diff
@@ -10,23 +10,20 @@ jobs:
     name: Build, test, and publish unstable release
     if: "! contains(github.event.head_commit.message, 'chore(release): publish')"
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        node-version: [14.x, 16.x]
```

Comment:
@BenLiuMattr does it mean we are no longer support 14.x ?

Reply:
we still need to support 14 and 16 (checked in `any-pr.yml` workflow), just for this 
 npm publish workflow, we should only allow one job to run the publish, otherwise, the latter one will fail for publishing the same version twice, also it does not make sense to publish it multiple times.

Reply:
but we do have some publish binary workflows that needs to build and publish in different Node versions. For example https://github.com/mattrglobal/node-bbs-signatures/blob/master/.github/workflows/push-release.yml#L9
But for npm package (ts only), we don't need this strategy.